process_preds.Rmd

```diff
@@ -74,11 +87,18 @@ process_preds <- function(predictor_path, template, resolution, distance=FALSE,
 
     # Set template temporarily to raster CRS
```

Comment:
I had to remind myself why we do it this way first, so could be helpful to add a comment here like "transform template to raster crs for faster initial cropping of large rasters"